service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/C2UploadedEventHandler.java

```diff
@@ -54,16 +54,15 @@ public void sendC2UploadedNotificationToAllocatedJudge(final C2UploadedEvent eve
         EventData eventData = new EventData(event);
         CaseData caseData = mapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
 
-        if (featureToggleService.isAllocatedJudgeNotificationEnabled(C2_APPLICATION)) {
-            if (caseData.allocatedJudgeExists()) {
-                AllocatedJudgeTemplateForC2 parameters = c2UploadedEmailContentProvider
-                    .buildC2UploadNotificationForAllocatedJudge(eventData.getCaseDetails());
+        if (featureToggleService.isAllocatedJudgeNotificationEnabled(C2_APPLICATION)
+            && caseData.allocatedJudgeExists()) {
+            AllocatedJudgeTemplateForC2 parameters = c2UploadedEmailContentProvider
+                .buildC2UploadNotificationForAllocatedJudge(eventData.getCaseDetails());
 
-                String email = caseData.getAllocatedJudge().getJudgeEmailAddress();
+            String email = caseData.getAllocatedJudge().getJudgeEmailAddress();
```

Comment:
does it not fail now if there is no email? I am not sure if i understood it right!

Reply:
Yep I had thought allocatedJudge when implemented had an email address but this is not the case. Further extending if conditions now.